recipes/devel/flex.yaml

```diff
@@ -5,8 +5,8 @@ metaEnvironment:
 
 checkoutSCM:
     scm: url
-    url: https://sourceforge.net/projects/flex/files/flex-${PKG_VERSION}.tar.gz
-    digestSHA1: "cfe10b5de4893ced356adc437e78018e715818c3"
+    url: ${SOURCEFORGE_MIRROR}/flex/flex-${PKG_VERSION}.tar.gz
+    digestSHA256: fc367e9deb570ee7fc7893ce5066c2dbdedf3cf27e93f2f39d1318703b028f0c
```

Comment:
Please do not change the digest algorithm of packages in this PR. It will rebuild major parts for no good reason. We can still switch to SHA256 whenever the package is updated.